src/main/java/duke/commands/Command.java

```diff
@@ -0,0 +1,29 @@
+package duke.commands;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * Represents a command that can be interpreted by the system
+ */
+public abstract class Command {
+
+    /**
+     * Executes the command
+     * @param tasks Current list of tasks
```

Comment:
Aside the previous comment where I suggested adding punctuation, you might also wish to consider putting a blank line between the description and the parameters.